config/crd/bases/nac.oadp.openshift.io_nonadminbackups.yaml

```diff
@@ -738,6 +738,150 @@ spec:
                       file in object storage.
                     type: integer
                 type: object
+              veleroObject:
```

Comment:
Reading design I though about this: creating a struct that contains all velero things
- name
- namespace
- status

called veleroObject but it could be veleroBackup and veleroRestore for NAB and NAR respectively

Reply:
applied change to the code. See if you like. If not, I can undo the change